### PR TITLE
Blog archive test

### PR DESCRIPTION
**Pull request by nathakits**
Added metadata for top feature blogpost
Replace blog page layout with blog-archive layout and deleted archive page
Removed test classes

### CODE REVIEW

File: templates/blog.jade [hunk 1]
```diff
@@ -5,40 +5,51 @@ block append title
...
+    - var a0 = articles[0]
+    - var a0_img_src = (a0.metadata.featured_img ? a0.metadata.featured_img : "https://placehold.it/800x600")
+
+    div.Background-img-top--blog.Background-img-cover.u-padBottom--3
+      div(class="Background-img--full-wh" style="background-image: url('"+a0_img_src+"');")
+      div.Site-content-container.Background-img-caption-container.u-textCenter
+        a(href=a0.url)
+          div.Background-header-text= a0.title
+          p.Background-img-caption= moment.utc(a0.date).format('YYYY MMM DD')
+        if a0.metadata.subtitle
```
Comment (willpatera) [inline templates/blog.jade]: @nathakits -- is there an indentation error here? shouldn't statements coming after `if` be indented?

Comment (nathakits) [inline templates/blog.jade]: @willpatera - On line 49-51, statements after `if` was not indented so i thought it didn't have to.Currently It doesn't seem to have any problems though but i guess indentation but be better.

Comment (willpatera) [inline templates/blog.jade]: @nathakits -- indent: see docs at: https://pugjs.org/language/conditionals.html

Comment (nathakits) [inline templates/blog.jade]: Line 18 should be indented back though. Right now it's under `href`.

**Pull request merged**